frontend/src/main/components/UCSBSubjects/UCSBSubjectForm.js

```diff
@@ -108,18 +107,19 @@ function UCSBSubjectForm({ initialUCSBSubject, submitAction, buttonLabel="Create
                 </Form.Control.Feedback>
             </Form.Group>
 
-            <Form.Group className="mb-3" >
+            <Form.Group className="mb-3">
                 <Form.Label htmlFor="inactive">Inactive</Form.Label>
                 <Form.Control
                     data-testid="UCSBSubjectForm-inactive"
                     id="inactive"
-                    type="boolean"
+                    type="text"
                     isInvalid={Boolean(errors.inactive)}
-                    {...register("inactive", { required: true, pattern: bool_regex })}
```

Comment:
I believe this is necessary in order to display that a "boolean is required" error message. Without this check, when you try to input something other than a boolean it gives you a 400 error instead of an error under the text box like it was before.
![image](https://user-images.githubusercontent.com/56096744/156041720-5e3f60f2-3d57-449b-90b4-bbcc437f8eae.png)

Reply:
This is what it was like before
![image](https://user-images.githubusercontent.com/56096744/156042234-e5a8c4ac-1bf6-456e-befb-6bdb1a15d36c.png)